src/controller/user.controller.js

```diff
@@ -0,0 +1,95 @@
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const users = await userService.getAll();
+
+  if (!users) {
+    res.send(users);
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(users);
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+
+  const user = await userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
```

Comment:
You are converting the id to a number using the unary plus operator. While this works, it could lead to unexpected results if the id is not a number. It would be safer to check if the id is a number before trying to convert it.